kafka_metric/monitor.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+
+# Get min value from conf.d/*.ini settings
+function get_config_setting() {
+  min=1
+  config_files=$(find conf.d/ -name "*.ini")
```

Comment:
This agent doesn't have multiple config files. We either need to add the configuration for that or change the script. 

Likely it's best to change this script for the time being.